packages/frontend/src/theme/components/index.ts

```diff
@@ -14,4 +15,5 @@ export const components = {
   Link,
   Infobox,
   Text,
+  SingleSelect,
 }
```

Comment:
Can we also add a theme for `Radio`. 2 changes after discussing with @pregnantboy, I'll check with stacey in office as well.

1. Remove flashing on hover due to gap between radio
2. Remove red outline on select

I think we can make it default styling instead of doing a variant.

```
import { radioAnatomy } from '@chakra-ui/anatomy'
import { createMultiStyleConfigHelpers } from '@chakra-ui/react'

const { defineMultiStyleConfig } = createMultiStyleConfigHelpers(
  radioAnatomy.keys,
)

export const Radio = defineMultiStyleConfig({
  baseStyle: {
    container: {
      _hover: {
        bg: 'interaction.muted.neutral.hover',
      },
      _focusWithin: {
        outline: 0,
      },
    },
  },
})

```

Reply:
wah ok thanksss i think Stacey mentioned about the red outline too, i think this is amazing, thanks boss